src/app/game-state/activity.service.ts

```diff
@@ -1920,6 +1923,71 @@ export class ActivityService {
       unlocked: false,
       skipApprenticeshipLevel: 0
     };
+    
+    this.CombatCultivation = {
+      level: 0,
+      name: ['Combat Cultivation'],
+      activityType: ActivityType.CombatCultivtion,
+      description: ['Focus on the development of your combat skill.'],
+      consequenceDescription: ['Uses 200 Stamina. A very advanced cultivation technique. Make sure you have killed some monster before attempt this, teaches you how to do combat more effectively.'],
+      consequence: [() => {
+        this.characterService.characterState.status.stamina.value -= 200;
+        
+        const gemGrade = this.inventoryService.consume('spiritGem') / 10;
+
+        if (gemGrade > 0) {
+          this.battleService.tickCounter += 10;
```

Comment:
I don't know if this belongs with the rest. I'm not opposed to having it as an activity, but I think it should be an activity of its own.

Reply:
my original intention is since this activity can be run every day with high building such as capital, so i add tick counter to make sure there is enough gem to be consumed.

I removed it and also make this activity now with mana cost. Feel free to edit the values, I am just trying to think what can be done with the monster gem according to your trello's to do.